mlem/cli/deployment.py

```diff
@@ -4,90 +4,154 @@
 from typer import Argument, Option, Typer
 
 from mlem.cli.apply import run_apply_remote
+from mlem.cli.declare import add_env_params_deployment
 from mlem.cli.main import (
     app,
     mlem_command,
     mlem_group,
+    mlem_group_callback,
     option_data_project,
     option_data_rev,
     option_external,
+    option_file_conf,
     option_index,
     option_json,
+    option_load,
     option_method,
+    option_model,
+    option_model_project,
+    option_model_rev,
     option_project,
     option_rev,
     option_target_project,
 )
-from mlem.core.base import parse_string_conf
+from mlem.cli.utils import (
+    for_each_impl,
+    lazy_class_docstring,
+    make_not_required,
+    wrap_build_error,
+)
+from mlem.core.base import build_mlem_object
 from mlem.core.data_type import DataAnalyzer
-from mlem.core.errors import DeploymentError
+from mlem.core.errors import DeploymentError, MlemObjectNotFound
 from mlem.core.metadata import load_meta
-from mlem.core.objects import MlemDeployment
+from mlem.core.objects import (
+    DeployState,
+    DeployStatus,
+    MlemDeployment,
+    MlemModel,
+)
 from mlem.ui import echo, no_echo, set_echo
 
 deployment = Typer(
     name="deployment",
-    help="Manage deployments",
+    help="A set of commands to set up and manage deployments",
     cls=mlem_group("runtime", aliases=["deploy"]),
 )
 app.add_typer(deployment)
 
+deploy_run = Typer(
```

Comment:
Using thread (Using thread to keep comments under control, this is unrelated to a specific line highlighed)
Can't build docker image with this branch:
```cli
$ mlem declare env docker --registry local --daemon.host localhost docker-builder.mlem
$ mlem build docker --file_conf docker-builder --model rf.mlem --image.name=rf-docker
not enough values to unpack (expected 2, got 1)
```
What's wrong? @mike0sv

Reply:
`--file_conf` is in `$ mlem build docker -h` for me. Don't see `--load` there.

Reply:
Looks like this wasn't fixed by your last PR @mike0sv. This is not that urgent though. @madhur-tandon, could you please TAL and fix what I've reported in the last message above?

Reply:
Thank you @madhur-tandon, indeed, those were issues with my installation. Could you please fix the last 2 items? They still appear for me.

Reply:
I've created an issue so we won't forgot addressing it later https://github.com/iterative/mlem/issues/441